syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -560,6 +607,7 @@ def _tanh_chebyshev(tensor, maxval: int = 6, terms: int = 32):
           where c_i is the ith Chebyshev series coefficient and P_i is ith polynomial.
         The approximation is truncated to +/-1 outside [-maxval, maxval].
         Args:
+            tensor (tensor): values where the tanh needs to be approximated
```

Comment:
typo this should be here :)

Reply:
Hmm..this is for ```_tanh_chebyshev``` (forgot to put it in the previous PR)